### ISSUE TITLE
More config loading options

### ISSUE THREAD

**Issue by ashketchupppp** [ashketchupppp/veff#5]
Currently we have a `config.py` file which is in the source code. Ideally we want the user to not have to venture into the `veff` directory so they don't have to look at any source code. There needs to be an alternative place for a configuration to be loaded.

We can do two things
1. Attempt to load `config.py` from the top level directory by default. At the moment we load it from `/veff`, so that can be moved a directory up.
2. Allow the user to pass a `--config` parameter to the main process that tells `__main__.py` where to get the config from. Ideally this would use `argparse` from Python's standard library

In the event of `config.py` not being found and `--config` is not passed then we should log an error saying that the config could not be found and we should create a `config.py` containing some defaults at the top-level, to get the user started.

**Comment by Dhravya** [comment]
So like a CLI right?

**Comment by ashketchupppp** [comment]
Yeah, a CLI. We only need the one `--config` parameter for the moment
